docroot/modules/custom/uiowa_core/src/Entity/NodeBundleBase.php

```diff
@@ -71,6 +72,19 @@ public function buildCard(array &$build) {
     if (isset($build['#cache']['keys'])) {
       unset($build['#cache']['keys']);
     }
+
+    // Indicate sticky content if part of view with sticky sort applied.
+    $view = $this->values['view'] ?? NULL;
+    if ($view instanceof ViewExecutable) {
+      $sorts = $view->getDisplay()->getOption('sorts');
+      if (!empty($sorts)) {
+        if (array_key_exists('sticky', $sorts) && $this->isSticky()) {
```

Comment:
```suggestion
    $view = $this?->values['view'] ?? NULL;
    if ($view instanceof ViewExecutable) {
      $sorts = $view?->getDisplay()?->getOption('sorts');
      if (!empty($sorts)) {
        if (array_key_exists('sticky', $sorts) && $this?->isSticky()) {
```

I think that this can be changed for better coding safety?

Otherwise, I think the way you are checking for sticky seems good to me as well, as there is no way that a card will be sticky outside of a view, so checking there is good!

Reply:
Using `?->` and `?? NULL` is redundant. `?->` assigns the value to NULL if it doesn't resolve to a value.

Reply:
I think it should live there. I think it is a fine place.